tasties_app/migrations/0004_create_recipes.py

```diff
@@ -0,0 +1,975 @@
+            (3, 1, 5),
+            (1, 4),
+            (5, 1, 4),
+            (2, 3),
+            (2, 3, 5),
+            (4, 4, 4, 3),
+            (4, 3, 5),
+            (4, 4, 2, 4, 2, 2),
+            (5, 4, 2),
+            (3, 4, 4, 3, 2, 4),
+            (3, 1, 4),
+            (4, 4, 2, 4, 2, 2),
+            (4, 3, 5),
+            (4, 4, 4, 3),
+            (5, 1, 4),
+            (4, 3, 5),
+            (4, 4, 2, 4, 2, 2),
+            (3, 4, 4, 3, 2, 4),
+            (4, 3, 5),
+            (4, 4, 4, 3),
+            (5, 1, 4),
+            (4, 3, 5),
+            (4, 4, 2, 4, 2, 2),
+            (3, 4, 4, 3, 2, 4),
+            (4, 3, 5),
+            (4, 4, 4, 3),
+            (5, 1, 4),
+            (4, 3, 5),
+            (4, 4, 2, 4, 2, 2),
+            (3, 4, 4, 3, 2, 4),
+            (4, 3, 5),
+            (4, 4, 4, 3),
+            (5, 1, 4),
+            (4, 3, 5),
+            (4, 4, 2, 4, 2, 2),
+            (3, 4, 4, 3, 2, 4),
+            (4, 3, 5),
+            (4, 4, 4, 3),
+            (5, 1, 4),
+            (4, 3, 5),
+            (4, 4, 2, 4, 2, 2),
+            (3, 4, 4, 3, 2, 4),
+            (4, 3, 5),
+            (4, 3, 5),
+            (4, 4, 4, 3),
+            (5, 1, 4),
+            (5, 4, 2),
+            (3, 4, 4, 3, 2, 4),
+            (3, 1, 4),
+        ]
+
+        categories_list = []
+
+        for cat_name_sublist in cat_name_lists:
+            cat_object_sublist = []
+            for category_name in cat_name_sublist:
+                try:
+                    category_object = Category.objects.get(category_name=category_name)
+                    cat_object_sublist.append(category_object)
+                except Exception:
+                    continue
+            categories_list.append(cat_object_sublist)
+
+        authors = [author_pool[i % len(author_pool)] for i in range(0, len(titles))]
+
+        for (
+            author,
+            title,
+            description,
+            direction,
+            minute,
+            picture,
+            ing_sublist,
+            cat_sublist,
+            rating_sublist,
+        ) in zip(
+            authors,
+            titles,
+            descriptions,
+            directions,
+            minutes,
+            pictures,
+            ingredient_list,
+            categories_list,
+            recipe_ratings,
+        ):
+            if len(authors) == 0:
+                author = default_author
+            if Recipe.objects.filter(title=title):
+                title = title + "(1)"
+            if len(rating_sublist) > len(author_pool):
+                rating_sublist = ()
+            with transaction.atomic():
+                recipe = Recipe(
+                    author_id=author,
+                    title=title,
+                    description=description,
+                    directions=direction,
+                    minutes_to_make=minute,
+                    recipe_picture="static/images/" + picture,
```

Comment:
In order to present the images you will need to remove the `static/` at the beginning of the string.
I tested it locally.
When you perform `{% static 'img_url' %}` it already contains the static folder path.

Reply:
> In order to present the images you will need to remove the `static/` at the beginning of the string. I tested it locally. When you perform `{% static 'img_url' %}` it already contains the static folder path.

Thanks for the heads up. Will adjust.